bundles/mailjet-mail-connector/src/FondOfOryx/Zed/MailjetMailConnector/MailjetMailConnectorDependencyProvider.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace FondOfOryx\Zed\MailjetMailConnector;
+
+use FondOfOryx\Zed\MailjetMailConnector\Business\Model\Provider\MailjetMailer;
+use Mailjet\Client as MailjetClient;
+use Spryker\Zed\Kernel\AbstractBundleDependencyProvider;
+use Spryker\Zed\Kernel\Container;
+use Spryker\Zed\MailExtension\Dependency\Plugin\MailProviderPluginInterface;
+
+/**
+ * @method \FondOfOryx\Zed\MailjetMailConnector\MailjetMailConnectorConfig getConfig()
+ */
+class MailjetMailConnectorDependencyProvider extends AbstractBundleDependencyProvider
+{
+    /**
+     * @var string
+     */
+    public const MAILER_PROVIDER_MAILJET = 'MailjetMailConnector:MAILER_PROVIDER_MAILJET';
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    public function provideBusinessLayerDependencies(Container $container): Container
+    {
+        $container = $this->addMailer($container);
+
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    public function provideCommunicationLayerDependencies(Container $container): Container
+    {
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function addMailer(Container $container): Container
+    {
+        $self = $this;
+
+        $container->set(static::MAILER_PROVIDER_MAILJET, static function () use ($self) {
+            return $self->getMailer();
+        });
+
+        return $container;
+    }
+
+    /**
+     * @return \Spryker\Zed\MailExtension\Dependency\Plugin\MailProviderPluginInterface
+     */
+    protected function getMailer(): MailProviderPluginInterface
+    {
+        return new MailjetMailer(
+            new MailjetClient(
```

Comment:
If the client supports additional config I would like to see some configuration for that as well